users/views.py

```diff
@@ -210,3 +211,41 @@ def get_context_data(self, **kwargs: Any) -> dict:
         context = super().get_context_data(**kwargs)
         context["is_activated"] = self.activate()
         return context
+
+
+@method_decorator(login_required, name="dispatch")
+@method_decorator(
+    check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name, CustomUser.UserType.MANAGER.name]),
+    name="dispatch",
+)
+class NotificationUserListView(ListView):
+    template_name = "user_notifications_list.html"
+    model = CustomUser
+
+    def get_queryset(self) -> QuerySet:
+        return CustomUser.objects.filter(projects__managers__pk=self.request.user.pk).distinct()
+
+    def get_context_data(self, *, _object_list: Any = None, **kwargs: Any) -> dict:
+        context_data = super().get_context_data(**kwargs)
+        users_queryset = self.get_queryset()
+        today = datetime.date.today()
+        users_days_without_report_dict = {}
+        for user in users_queryset:
+            days_without_report_list = []
+            latest_report = user.report_set.order_by("-date").first()
+
+            if latest_report is not None:
+                latest_report_date = latest_report.date
+
+                for days_delta in range(1, (today - latest_report_date).days):
+                    no_report_day = latest_report_date + datetime.timedelta(days=days_delta)
+
+                    if no_report_day.weekday() not in [5, 6]:
```

Comment:
just one suggestion here:
maybe use `.isoweek()` and add comment, that Saturday and Sunday should be skipped

Reply:
Sure, changed :)